tests/playwright/src/utility/operations.ts

```diff
@@ -205,16 +205,30 @@ export async function deletePodmanMachine(page: Page, machineVisibleName: string
     if ((await podmanResourceCard.resourceElementConnectionStatus.innerText()) === ResourceElementState.Running) {
       await podmanResourceCard.performConnectionAction(ResourceElementActions.Stop);
       await playExpect(podmanResourceCard.resourceElementConnectionStatus).toHaveText(ResourceElementState.Off, {
-        timeout: 30_000,
+        timeout: 100_000,
       });
     }
     await podmanResourceCard.performConnectionAction(ResourceElementActions.Delete);
     await playExpect(podmanResourceCard.resourceElement).toBeHidden({ timeout: 30_000 });
+    await delay(10000);
+    await handleResetDefaultConnectionDialog(page);
   } else {
     console.log(`Podman machine [${machineVisibleName}] not present, skipping deletion.`);
   }
 }
 
+export async function handleResetDefaultConnectionDialog(page: Page): Promise<void> {
+  const connectionDialog = page.getByRole('dialog', { name: 'Podman' });
+  if (await connectionDialog.isVisible()) {
```

Comment:
What happens if isVisible() returns false because it's checked before the popup is displayed?